Makefile

```diff
@@ -66,7 +66,7 @@ test: check-llvm needs-cairo2 build-alexandria runtime-ci
 	cargo test --profile ci --features=scarb,with-cheatcode,with-debug-utils
 
 .PHONY: test-cairo
-test-cairo: check-llvm needs-cairo2 build-alexandria runtime-ci
```

Comment:
Why removing the alexandría build? Does their PR updating cairo-lang work?

Reply:
i didnt remove it, just from test-cairo, which tests the corelib, so its not needed for this target

Reply:
Oh okay